upgrade/Validate_CSM_Health_During_Upgrade.md

```diff
@@ -4,7 +4,9 @@
   since the CSM services were upgraded. This allows the various Kubernetes resources to
   initialize and start.
 - Although it is not recommended, the [Booting CSM `barebones` image](../operations/validate_csm_health.md#5-booting-csm-barebones-image)
-  test may be skipped if all compute nodes are active running application workloads.
+  test may be skipped if all compute nodes are active running application workloads. In CSM 1.6.0, a known issue
+  causes a [Barebones Boot Test Failure](../troubleshooting/known_issues/barebones_boot_test_failure.md).
+  This issue is resolved in CSM 1.6.1.
```

Comment:
Draw more attention to this by moving it from the end of a series of statements to nesting it underneath in its own blockquote.
```suggestion
  > This test is broken in CSM 1.6.0 and should be skipped for that release. For more information, see the known issue, [Barebones Boot Test Failure](../troubleshooting/known_issues/barebones_boot_test_failure.md).
```